modules/system/controllers/Settings.php

```diff
@@ -187,4 +217,23 @@ protected function findSettingItem($author, $plugin, $code)
 
         return $item;
     }
+
+    /**
+     * Auto-guesses the requested setting item by URL segments from the Request object.
+     *
+     * @return array
+     */
+    protected function autoGuessSettingItem()
```

Comment:
```suggestion
    protected function guessSettingItem()
```
Adding "auto" to this method name seems redundant

Reply:
True - changed